README.md

```diff
@@ -12,7 +12,7 @@ download bank transactions.
 
 ## Getting Started
 
-1. create a example config `bank-downloader init`
+1. create a example config and download the playwright patched browsers with `bank-downloader init`
```

Comment:
playwright isn't used anymore since they only support debian or ubuntu. Which means i can only test and release this in docker.

So I switched to `chromedp` as that just leans on the chrome you have installed.